pkg/scheduler/framework/events.go

```diff
@@ -145,7 +175,7 @@ var (
 
 // PodSchedulingPropertiesChange interprets the update of a pod and returns corresponding UpdatePodXYZ event(s).
 // Once we have other pod update events, we should update here as well.
-func PodSchedulingPropertiesChange(newPod *v1.Pod, oldPod *v1.Pod) (events []ClusterEvent) {
+func PodSchedulingPropertiesChange(newPod *v1.Pod, oldPod *v1.Pod, isSelf bool) (events []ClusterEvent) {
```

Comment:
Rather than adding a new param `isSelf`, can we just determine `isSelf` by comparing the UIDs of newPod and oldPod?

Reply:
Actually I think we can't?

This part seems a bit awkward. I believe that the UID of newPod and oldPod should always be the same (isn't that right?).

The`isSelf` is used in scenarios like this: 
When podA is updated, we attempt to use an event to reschedule podB in the queue. Without `isSelf`, `PodSchedulingPropertiesChange(newPodA, oldPodA)` would return `PodItselfUpdate`. However, since we are trying to reschedule another pod, we expect it to return `UnscheduledPodUpdate` instead.

Reply:
Ah yes. Sorry, I mixed up